index.html

```diff
@@ -4,191 +4,204 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Portfolio-Mobile</title>
+    <title>Portfolio</title>
     <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css">
 </head>
 <body>
     <!-- the header section-->
-    <header>
+    <header class="header-section">
         <nav class="navbar">
-            <a class="logo" href="https://github.com/chibunduonyeje1">SO</a>
-            <div class="hamburger"><img src="./images/hamburger.png" alt=""></div>
+            <a class="logo" href="https://github.com/chibunduonyeje1">Stephen Onyeje</a>
+            <div class="menu">
+                <ul class="menu-nav">
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                    <a href="#"><i class="fa-regular fa-envelope" ></i></a>
+                </ul>
+                <img src="./images/hamburger.png" alt="">
+            </div>
         </nav>
     </header>
 
     <!-- the hero section -->
-    <section>
-        <div class="herosection">
-            <h1>Hey There.<br>I’m Stephen Onyeje</h1>
-            <h2>I’am a Software Developer</h2>
-            <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-            <div class="social-icon">
-                <ul>
-                    <li><a href="#"><img src="./images/Vector1.png" alt=""></a></li>
-                    <li><a href="#"><img src="./images/Vector2.png" alt=""></a></li>
-                    <li><a href="#"><img src="./images/Vector4.png" alt=""></a></li>
-                    <li><a href="#"><img src="./images/Vector5.png" alt=""></a></li>
-                    <li><a href="#"><img src="./images/Vector6.png" alt=""></a></li>
-                </ul>
+    <section class="herosection-wrapper">
+        <section class="herosection">
+            <div class="herosection-text">
+                <h1>Hey There.<br>I’m Stephen Onyeje</h1>
+                <h2>I’am a Software Developer</h2>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
Kindly make sure you follow the color, fonts, padding etc as we have it in the Figma design ⚔️😉.
| Your Design | Figma Design |
| :---:   | :---: | 
| <img width="840" alt="image" src="https://user-images.githubusercontent.com/73854999/206789321-64524b66-4f33-430e-abd5-d4e0a623d711.png">|<img width="840" alt="image" src="https://user-images.githubusercontent.com/73854999/206789450-a671a639-32e3-4105-8c66-104e100ef11b.png">|